Gruntfile.js

```diff
@@ -5,9 +5,13 @@ module.exports = function(grunt) {
   grunt.initConfig({
     pkg : grunt.file.readJSON('package.json'),
 
-    src_dir_js : [ 'src/main/resources/public/js' ],
+    build_dir_js: 'build/resources/main/public/js',
 
-    build_dir_js : [ 'build/resources/main/public/js' ],
+    gruntfile_js: 'Gruntfile.js',
+
+    src_app_js: 'src/main/resources/public/js/app.js',
+
+    src_files_js: 'src/main/resources/public/js/*.js',
```

Comment:
I checked this and found out that this was wrong so ich add ``/**`` again. Also removed the ``*`` from the jshint dist src.